packages/pages/src/components/link/link.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { Link } from "./link";
+
+describe("Link", () => {
+  it("renders component when given href + children", () => {
+    render(<Link href="https://yext.com">Click Me</Link>);
+  });
+
+  it("renders component when given partial cta and children", () => {
+    render(<Link cta={{ link: "https://yext.com" }}>Click Me</Link>);
+  });
+
+  it("renders component when given full cta prop and no children", () => {
+    render(<Link cta={{ link: "https://yext.com", label: "Click Me" }} />);
+  });
+
+  it("renders component when given full cta prop and no children", () => {
```

Comment:
Partial props?

Reply:
sounds like this is resolved?

Reply:
@mkilpatrick I'm going to resolve this unless you were talking about something else